FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m

```diff
@@ -129,4 +203,58 @@ - (void)deliverNotification {
   }
 }
 
+- (void)exportDeliveryMetricsToBigQueryWithMessageInfo:(NSDictionary *)info {
+  GDTCORTransport *transport = [[GDTCORTransport alloc] initWithMappingID:@"1249"
+                                                             transformers:nil
+                                                                   target:kGDTCORTargetFLL];
+
+  fm_MessagingClientEventExtension eventExtension = fm_MessagingClientEventExtension_init_default;
```

Comment:
optional: It seems that refactoring out the creating the event payload to a separate method can make the method more readable, e.g.:
```
FIRMessagingMetricsLog *log = [self logWithMessageInfo:info]; // The message composing should be moved there.

GDTCOREvent *event = [transport eventForTransport];
event.dataObject = log;
event.qosTier = GDTCOREventQoSFast;

[transport sendDataEvent:event];
```

Reply:
I tried to refactor in this way and then receiving message crash. I'm guessing sth not get copied properly inside FIRMessagingMetricsLog object. So i'm not gonna submit with this change and we can figure out later.